custom-words.txt

```diff
@@ -1329,6 +1329,7 @@ kafkarest
 kaliningrad
 kangri
 karachay
+karpenter
```

Comment:
please create a separate PR to update the custom-words

Since the review cycle is very long and other people will also modify this file, it is very common to meet a merge conflict, and it is very troublesome to resolve the conflict for the dev branch.

Reply:
Done, and removed here.

See PR: https://github.com/Azure/azure-rest-api-specs/pull/26135

Do you know who I need to track down to let it merge?